interface/patient_file/encounter/forms.php

```diff
@@ -20,6 +20,10 @@
     $groupId = $therapy_group;
 }
 $attendant_id = $attendant_type == 'pid' ? $pid : $therapy_group;
+if(!(acl_check("groups","glog",false, 'view') OR acl_check("groups","glog",false, 'write'))){
```

Comment:
fixed, 
'glog' is for group encounter view/write mode. 
this way we lock the user from viewing and using the  encounter log screen.